cmd/clusterctl/client/client.go

```diff
@@ -57,6 +57,7 @@ type Client interface {
 	// PlanUpgrade returns a set of suggested Upgrade plans for the cluster, and more specifically:
 	// - Upgrade to the latest version in the v1alpha3 series: ....
```

Comment:
I wonder how useful this list of contracts here is and if we'll be able to keep this up-to-date in the future (as we didn't in the past)

(similar in other places)

Reply:
Sure - probably better off just removing this.